_layouts/blog-list.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: slim
+---
+
+<section class="mb-5 py-3">
+  <div class="container">
```

Comment:
These container, row and col `div`s can be dropped completely, the slim layout forces the narrow column width for the whole page.

Reply:
Thanks, I've removed this.